docs/en/user_guides/prepare_datasets.md

```diff
@@ -80,7 +93,7 @@ An example of the dataset config is as follows.
 4. `type`: 'upper' or 'lower', will be used in data augmentation.
 5. `swap`: indicates the 'swap pair' (also known as 'flip pair'). When applying image horizontal flip, the left part will become the right part. We need to flip the keypoints accordingly.
 
-`skeleton_info` contains the information about the keypoint connectivity, which is used for visualization.
+`skeleton_info` contains information about the keypoint connectivity, which is used for visualization.
 
 `joint_weights` assigns different loss weights to different keypoints.
 
```

Comment:
```Python
keypoint_info={
        0:
        dict(name='nose', id=0, color=[51, 153, 255], type='upper', swap=''),
        1:
        dict(
            name='left_eye',
            id=1,
            color=[51, 153, 255],
            type='upper',
            swap='right_eye'),
        2:
        dict(
            name='right_eye',
            id=2,
            color=[51, 153, 255],
            type='upper',
            swap='left_eye'),
        3:
        dict(
            name='left_ear',
            id=3,
            color=[51, 153, 255],
            type='upper',
            swap='right_ear'),
```

I think this list is too long to read and it't not necessary to show the whole `keypoint_info` list in this example

Reply:
Agreed. I think we can omit the middle part of `keypoint_info` as well as `skeleton_info`.

Reply:
Good idea